### PR TITLE
AKS cluster scans - Handled error and removed $env variables

### PR DESCRIPTION
**Pull request by TarunKrShukla**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 [hunk 1]
```diff
@@ -41,17 +41,27 @@ class KubernetesClusterCA : AzCommandBase {
         {
             # This method verifies that if both Azure CLI and Kubernetes CLI are installed or not
             $IsPrerequisitesPresent = $false
-            $azCliCmdOutput  = az --help --ouput json
-            $aksliCmdOuput = kubectl help
+            try{
+                $azCliCmdOutput  = az --help --ouput json
```
Comment (abhaydaga) [inline src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Is it --ouput or --output??

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 [hunk 2]
```diff
@@ -100,7 +110,8 @@ class KubernetesClusterCA : AzCommandBase {
             
             # Download deployment file from server and store it in temp location
             $deploymentFileUrl =  $this.deploymentFileBaseUrl + $this.deploymentFileName
-            $filePath = Join-Path $($env:TEMP)  $($this.deploymentFileName)
+            $InClusterCATempFolderPath = Join-Path $([Constants]::AzSKTempFolderPath) "InClusterCA";
```
Comment (abhaydaga) [inline src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: "InClusterCA" can be moved as a variable in Constants.ps1

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 [hunk 3]
```diff
@@ -199,8 +210,7 @@ class KubernetesClusterCA : AzCommandBase {
                         if($jobPodsCount -gt 0){
                             $moduleName = $this.GetModuleName();
                             $this.PublishCustomMessage("Downloading existing job's log to local machine..." , [MessageType]::Warning)
-                            $baseFolder = Join-Path $($Env:LOCALAPPDATA) "Microsoft" | Join-Path -ChildPath $($moduleName) | Join-Path -ChildPath "Logs\ClusterScans\Kubernetes"| Join-Path -ChildPath $($this.SubscriptionContext.SubscriptionId )| Join-Path -ChildPath $($this.ResourceGroupName)| Join-Path -ChildPath $($this.ResourceName)
-                            #$baseFolder =  $Env:LOCALAPPDATA + "\Microsoft\" + $moduleName + "Logs\ClusterScans\Kubernetes\"+ $this.SubscriptionContext.SubscriptionId +"\"+  $this.ResourceGroupName +"\"+ $this.ResourceName+ "\" ; 
+                            $baseFolder = Join-Path $([Constants]::AzSKAppFolderPath) "Logs" | Join-Path -ChildPath "ClusterScans\Kubernetes"| Join-Path -ChildPath $($this.SubscriptionContext.SubscriptionId )| Join-Path -ChildPath $($this.ResourceGroupName)| Join-Path -ChildPath $($this.ResourceName)
```
Comment (abhaydaga) [inline src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Shouldn't we move hard-coded paths to Constants.ps1?

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 [hunk 4]
```diff
@@ -401,7 +414,8 @@ class KubernetesClusterCA : AzCommandBase {
             
                 # Download deployment file from server and store it in temp location
                 $deploymentFileUrl =  $this.deploymentFileBaseUrl +  $this.configMapFileName
-                $filePath = Join-Path $($env:TEMP) $this.configMapFileName
+                $InClusterCATempFolderPath = Join-Path $([Constants]::AzSKTempFolderPath) "InClusterCA";
```
Comment (abhaydaga) [inline src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Can be moved to COnstants.ps1

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 [hunk 5]
```diff
@@ -422,7 +436,8 @@ class KubernetesClusterCA : AzCommandBase {
         
                 # Download deployment file from server and store it in temp location
                 $deploymentFileUrl =  $this.deploymentFileBaseUrl +  $this.runtimeAccountFileName
-                $filePath = Join-Path $($env:TEMP) $this.runtimeAccountFileName
+                $InClusterCATempFolderPath = Join-Path $([Constants]::AzSKTempFolderPath) "InClusterCA";
```
Comment (abhaydaga) [inline src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Can be moved to Constants.ps1

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 [hunk 6]
```diff
@@ -462,7 +477,8 @@ class KubernetesClusterCA : AzCommandBase {
             
                     # Download deployment file from server and store it in temp location
                     $deploymentFileUrl =  $this.deploymentFileBaseUrl +  $this.jobSpecificationsFileName
-                    $filePath = Join-Path $($env:TEMP) $this.jobSpecificationsFileName
+                    $InClusterCATempFolderPath = Join-Path $([Constants]::AzSKTempFolderPath) "InClusterCA";
```
Comment (abhaydaga) [inline src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Can be moved to Constants.ps1

Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Will do in later builda

File: src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1 (hunk 4 above)
Comment (TarunKrShukla) [reply src/AzSK/Framework/Core/InClusterCA/ContinuousAssuranceForKubernetes.ps1]: Will do in later build

**Pull request merged**